python.d/dockerd.chart.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Description: docker netdata python.d module
+# Author: Kévin Darcel (@tuxity)
+
+try:
+    import docker
+    DOCKER_PY = True
+except ImportError:
+    DOCKER_PY = False
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+# default module values (can be overridden per job in `config`)
+# update_every = 1
+priority = 60000
+retries = 60
+
+# charts order (can be overridden if you want less charts, or different order)
+ORDER = [
+    'running_containers',
+    'healthy_containers',
+    'unhealthy_containers'
+]
+
+CHARTS = {
+    'running_containers': {
+        'options': [None, 'Number of running containers', 'running containers', 'running containers', 'docker.running_containers', 'line'],
+        'lines': [
+            ['running_containers', 'running']
+        ]
+    },
+    'healthy_containers': {
+        'options': [None, 'Number of healthy containers', 'healthy containers', 'healthy containers', 'docker.healthy_containers', 'line'],
+        'lines': [
+            ['healthy_containers', 'healthy']
+        ]
+    },
+    'unhealthy_containers': {
+        'options': [None, 'Number of unhealthy containers', 'unhealthy containers', 'unhealthy containers', 'docker.unhealthy_containers', 'line'],
+        'lines': [
+            ['unhealthy_containers', 'unhealthy']
+        ]
+    }
+}
+
+
+class Service(SimpleService):
+    def __init__(self, configuration=None, name=None):
+        SimpleService.__init__(self, configuration=configuration, name=name)
+        self.order = ORDER
+        self.definitions = CHARTS
+
+    def check(self):
+        if not DOCKER_PY:
+            self.error('\'docker\' package is needed to use docker.chart.py')
+            return False
+
+        self.client = docker.DockerClient(base_url=self.configuration.get('url', 'unix://var/run/docker.sock'))
+
+        return True if self.client else False
+
+    def get_data(self):
+        data = dict()
+        data['running_containers'] = len(self.client.containers.list())
+        data['healthy_containers'] = len(self.client.containers.list(filters={'health': 'healthy'}))
+        data['unhealthy_containers'] = len(self.client.containers.list(filters={'health': 'unhealthy'}))
+
+        return data or None
```

Comment:
`self.client.containers.list()` returns a list of `Containers` objects. Does object provide method/attribute about health status?

If so we can call `self.client.containers.list()` only once then.

Reply:
It doesn't seems possible, we have to call it multiple times https://docker-py.readthedocs.io/en/stable/containers.html

Reply:
Oh yes we can do that, I'm adding it